src/integrationTest/resources/application-integration.yaml

```diff
@@ -57,3 +57,13 @@ service-config:
       transformation-url: *wiremock
       case-type-ids:
         - Bulk_Scanned
+      form-type-to-surname-ocr-field-mappings:
+        - form-field-mapping:
```

Comment:
How about not having this root field and just have `formType` and `ocrField` under each element?

Reply:
What do you mean? Something like this:
form-type-to-surname-ocr-field-mappings:
  - form1 field1
  - form2 field2
?
I think existing config is better

Reply:
This is what I'm proposing:

```
   form-type-to-surname-ocr-field-mappings:
     - formType: form1
        ocrField: lastName
     - formType: form2
        ocrField: surname
     - formType: form3 // not form 2 btw
        ocrField: nickName
```

Reply:
Will do. Btw it is deliberately form2 - to ensure that multiple mappings in yaml config is handled properly

Reply:
I thought it could be the reason, but now the app won't even start if we don't have correct configuration. At least it's not expected to.

Reply:
Somehow my comment became outdated, so I'm putting it here:
> The app should fail to start if we have multiple mapping for the same jurisdiction + form type. The error log is about finding multiple such keys in the actual data.

Reply:
This is a requirement - not to fail